spec.html

```diff
@@ -24798,6 +24836,7 @@ <h2>Syntax</h2>
         ExportSpecifier :
           IdentifierName
           IdentifierName `as` IdentifierName
+          IdentifierName `as` ModuleExportName
```

Comment:
The following change will address both <https://github.com/tc39/ecma262/pull/2154#issuecomment-696362312> and <https://github.com/tc39/ecma262/pull/2154#issuecomment-696379292>:

```suggestion
        ExportSpecifier :
          IdentifierName
          IdentifierName `as` IdentifierName
          IdentifierName `as` ModuleExportName
          ModuleExportName
          ModuleExportName `as` ModuleExportName
```

---

Also, remember to add an early error for when the following is not a part of an `ExportFromClause`:

```emu-grammar
ExportSpecifier :
  ModuleExportName
  ModuleExportName `as` ModuleExportName
```